handler.go

```diff
@@ -103,7 +103,8 @@ func handle(out chan string, m *message) {
 		for _, prefix := range cmdPrefixes {
 			if strings.HasPrefix(msg, prefix) {
 				p := split(m.Args[1][len(prefix):], 2)
-				if c := cmds.Get(p[0]); c != nil && (!c.modOnly || m.Mod) {
+				isMod := m.Mod || strings.ToLower(m.DisplayName) == CHANNEL
```

Comment:
Display names can differ from login for Asian users as well as special channels like Riot Games. The easiest safe way to check afaik is to see if the "user-id" and "room-id" tags match